src/components/MerchantsPage/gam/CampaignListItem.tsx

```diff
@@ -81,7 +81,9 @@ const CampaignListItem = (props: Props) => {
             window.location.href = '/' + merchant.seller_id;
           }}>Visit merchant</Button>
```

Comment:
where's the logic to turn this button black from red if the campaign is inactive? I might just be missing it

Reply:
Ooops thats not a feature...hovering over the button turns it red. I just happened to be doing this when I took the screenshot 😂

Reply:
Ahhh gotchya, that makes sense, thanks for clarifying! 😂